index.html

```diff
@@ -8537,22 +8556,24 @@ <h3 id="Nature"><span class="wip">Nature - Use with Caution</span></h3>
                </td>
                <td>Tree of Life
                </td>
-               <td>Heal all allies within 25ft as much as you decide, then suffer half the
+               <td>Heal all allies within 20ft as much as you decide, then suffer half the
```

Comment:
I am interested to see if this is real strong or not - but it's cool so it gets a go ahead from me

Reply:
At 2AP it was far too easy to combo with temp HP and be very strong.  I think 3AP and a shorter range makes it quite hard to pull off, so as a high ability I'm not too worried.